demo/full/styles/style.css

```diff
@@ -341,38 +391,128 @@ body {
   margin-top: -7px;
 }
 
+.player-knobs-header {
+  padding: 9px 20px 9px 20px;
+  border-bottom: 1px dotted #555;
+}
+
 .player-knobs {
-  margin: 0px auto 20px auto;
-  text-align: left;
+  width: 30%;
+  max-width: 350px;
+  min-width: 300px;
+  color: white;
+  position: absolute;
+  bottom: 10px;
+  right: 10px;
+  background-color: rgba(0,0,0);
+  opacity: 0;
+  transform: scale(0.9);
+  transition: opacity 200ms;
+  z-index: 100;
+  pointer-events:none;
+}
+
+.player-knobs.fade-in-out {
+  transform: scale(1);
+  pointer-events: auto;
+  opacity: 0.8;
+}
+
+.player-knobs-title {
+  font-weight: bold;
+}
+
+.player-knobs-close {
+  font-family: "icons", sans-serif;
+  color: #d0d0cf;
+  font-size: 14px;
+  margin-top: auto;
+  margin-bottom: auto;
+  cursor: pointer;
+}
+
+.player-knobs-close:hover {
+  color: #ffffff;
+}
+
+.player-knobs-knobs {
+  padding: 10px 20px 10px 20px;
+  display: flex;
+  flex-direction: column;
+}
+
+.player-knobs > div {
+  display: flex;
+  justify-content: space-between;
 }
 
 .knob {
-  display: inline-block;
-  margin: 10px;
+  display: flex;
+  margin: 5px 0 5px 0;
 }
 
 .knob-name {
-  display: inline-block;
   margin-right: 10px;
+  overflow: hidden;
+  white-space: nowrap;
+  flex: 1;
+  margin: auto;
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Unexpected shorthand "margin" after "margin-right" ([css:S4657](https://sonarcloud.io/organizations/rx-player/rules#rule_key=css%3AS4657))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=rx-player&pullRequest=312&issues=AWXYTOxW_eEZPtUmDdcd&open=AWXYTOxW_eEZPtUmDdcd)

[ISSUE_KEY]: AWXYTOxW_eEZPtUmDdcd